### ISSUE TITLE
Adicionar dados de 2021

### ISSUE THREAD

**Issue by gabubellon** [gabubellon/pybr_opendata#8]

**Comment by dandaramcsousa** [comment]
aqui é só mudar no `util.py` né para quando estivermos os dados de 2021 né?

**Comment by gabubellon** [comment]
@dandaramcsousa sim, mas vamos ter de deixar generico pra 2020 e 2021, acho que adicionar um slug {ano} ali e pegar de um filtro já ajuda
